eventdata/parameter_sources/elasticlogs_kibana_source.py

```diff
@@ -164,7 +169,11 @@ def params(self):
             offset = (int)(random.random() * math.fabs(t2 - t1))
 
             ts_max_ms = int(offset + min(t1, t2))
-        
+
+        if self._window_length == "random":
+            max_window_length = int(math.fabs(self._fieldstats_end_ms - self._fieldstats_start_ms))
+            self._window_duration_ms = random.randrange(int(60*1000), max_window_length, int(60*1000))
```

Comment:
`randrange` does not include the stop (`max_window_length`), so it'll never include the value `max_window_length`. Do we want this behavior?

Reply:
We don't need `int` here, `60*1000` is integer.

Reply:
> `randrange` does not include the stop (`max_window_length`), so it'll never include the value `max_window_length`. Do we want this behavior?

Right, I can change it to include it.